scripts/upgrade

```diff
@@ -161,11 +271,28 @@ ynh_exec_warn_less ynh_install_app_dependencies $pkg_dependencies
 #=================================================
 ynh_script_progression --message="Updating a configuration file..." --weight=2
 
+# reset permissions to be able to apply_permissions with app_setting values after upgrade
+listrelay_=$listrelay
+listuser_=$listuser
+listadmin_=$listadmin
+listrelay="*"
+listuser="@user:domain.tld"
+listadmin="@admin:domain.tld"
+
 ynh_add_config --template="../conf/config.yaml" --destination="$final_path/config.yaml"
 
 chmod 400 "$final_path/config.yaml"
 chown $app:$app "$final_path/config.yaml"
 
+listrelay=$listrelay_
+listuser=$listuser_
+listadmin=$listadmin_
+
+# apply_permissions to have correct syntax in config file
+yunohost app config set $app main.permissions.listrelay -v "$listrelay"
```

Comment:
I've been checking why the CI fails on `upgrade` and it seems that the issue may be related to these lines...
As far as I understood when it calls `yunohost app config set $app main.permissions.listrelay -v "$listrelay"` it is called from the old version of the app (the config panel doesn't exist yet) and so it fails, throws a warning and stops the upgrade process.

I'm not sure how to solve it for the moment, `yq` would be very helpful here, I need to check if it would be handled soon to the Yuno dependencies because they mostly agreed it was fine to add it 🙂  if not I'll have to find an alternative solution for this.